client/src/main/python/ss-module-download.py

```diff
@@ -72,16 +84,53 @@ def _removeRuns(self, root):
             for run in runs.findall('*'):
                 runs.remove(run)
 
-    def _removeCloudImageIdentifiers(self, root):
-        """Remove the cloudImageIdentifiers element from
-           the given document.  These identifiers are not
-           portable between SlipStream deployments."""
+    def _remove_clouds(self, root):
+        """Remove the cloudImageIdentifiers, cloudNames and cloud specific parameters element from the given document.
+           These elements are not portable between SlipStream deployments."""
 
         ids = root.find('cloudImageIdentifiers')
         if ids is not None:
             for id in ids.findall('*'):
                 ids.remove(id)
 
+        cloud_names = root.find('cloudNames')
+        if cloud_names is not None:
+            parameters = root.find('parameters')
+            if parameters is not None:
+                for cloud_name in cloud_names.findall('*'):
+                    cloud_parameters = parameters.findall("./entry/parameter[@category='%s'].." % cloud_name.text)
```

Comment:
I don't have a Python 2.6 to test but I didn't find anything in the documentation which could let me think it will not work.
In the [Supported XPath syntax](https://docs.python.org/2/library/xml.etree.elementtree.html?highlight=findall#supported-xpath-syntax) there is no mention of Python 2.6.
And the method [`findall(match)`](https://docs.python.org/2/library/xml.etree.elementtree.html?highlight=findall#xml.etree.ElementTree.Element.findall) doesn't require Python 2.7.